python/ql/src/Security/CWE-079/Jinja2WithoutEscaping.qhelp

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+
+Cross-site scripting attacks can occur if untrusted input is not escaped. This applies to templates as well as code.
```

Comment:
Cross-site scripting (XSS)

Please add the short form here since we use it later.